static/app/views/replays/detail/userActionsNavigator.tsx

```diff
@@ -1,42 +1,94 @@
 import React from 'react';
+import styled from '@emotion/styled';
 
-import EventEntry from 'sentry/components/events/eventEntry';
-import {Entry, Event} from 'sentry/types/event';
-import useOrganization from 'sentry/utils/useOrganization';
-import {useRouteContext} from 'sentry/utils/useRouteContext';
+import Type from 'sentry/components/events/interfaces/breadcrumbs/breadcrumb/type';
+import {
+  filterCrumbs,
+  transformCrumbs,
+} from 'sentry/components/events/interfaces/breadcrumbs/utils';
+import {
+  Panel as BasePanel,
+  PanelBody as BasePanelBody,
+  PanelHeader as BasePanelHeader,
+  PanelItem,
+} from 'sentry/components/panels';
+import ActionCategory from 'sentry/components/replays/actionCategory';
+import PlayerRelativeTime from 'sentry/components/replays/playerRelativeTime';
+import space from 'sentry/styles/space';
+import {RawCrumb} from 'sentry/types/breadcrumbs';
+import {Event, EventTransaction} from 'sentry/types/event';
 
 type Props = {
-  entry: Entry | undefined;
+  crumbs: RawCrumb[];
   event: Event | undefined;
 };
 
-function UserActionsNavigator({event, entry}: Props) {
-  // TODO(replay): New User Actions widget replaces this breadcrumb view
-
-  const {routes, router} = useRouteContext();
-  const organization = useOrganization();
-
-  if (!entry || !event) {
+function UserActionsNavigator({event, crumbs}: Props) {
+  if (!event) {
     return null;
   }
 
-  const projectSlug = getProjectSlug(event);
+  const relativeTime = (event as EventTransaction).startTimestamp;
```

Comment:
Could update the prop type instead of using `as EventTransaction`. That'll put typescript to work if there are any errors we need to know about